### ISSUE TITLE
Arrow representation can fail to invert using keyboard navigation

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/geometric-optics#331]
Seen testing master as part of #328 testing on Windows 10, Chrome

Steps to reproduce:
1. Tab to the arrow representation
2. Using the keyboard arrow keys to step down or up to invert the arrow represenation
3. Single presses can cause this behavior (does not always occur), but a continuous hold of the keys does not seem to trigger the issue

See the video below (I show the behavior using single presses of the key and then "moving through" using a "fire on hold" with the key:

https://user-images.githubusercontent.com/4051174/155824880-29eb431c-034e-4aea-9efa-03625231093b.mp4

**Comment by pixelzoom** [comment]
Fixed in master.   With arrow key held down, it will behave like mouse/touch, and snap to minimum magnitude when you release the key.  If you press and release arrow key (don't hold it down), when the arrow is less than minimum magntude, it will snap to minimum magnitude with the sign inverted.

@ariel-phet please review in master, closed if OK.

**Comment by ariel-phet** [comment]
@pixelzoom appears fixed in master, closing.
